website/docs/Pipelines/Whole_Genome_Germline_Single_Sample_Pipeline/README.md

```diff
@@ -303,9 +303,61 @@ The table below describes the final workflow outputs. If running the workflow on
 | output_cram_index | Index for the aligned recalibrated CRAM. | File |
 | output_cram_md5 | MD5 checksum for the aligned recalibrated BAM. | File |
 | validate_cram_file_report | Validated report for the CRAM created with the ValidateSam tool.  | File |
-| output_vcf | Final VCF with variant calls produced by HaplotypeCaller. | File |
-| output_vcf_index | Index for the final VCF. | File |
+| output_vcf | Final [reblocked](https://gatk.broadinstitute.org/hc/en-us/articles/360037593171) GVCF with variant calls produced by HaplotypeCaller (read more in the [Reblocking](#reblocking) section below).| File |
+| output_vcf_index | Index for the final GVCF. | File |
  
+### Reblocking
+Reblocking is a process that compresses a HaplotypeCaller GVCF by merging homRef blocks according to new genotype quality (GQ) bands. 
+
+As of November 2021, reblocking is a default task in the WGS pipeline. To skip reblocking, add the following to the workflow's input configuration file (JSON):
+
+```WDL
+"WholeGenomeGermlineSingleSample.BamToGvcf.skip_reblocking": true
+```
+
+The [Reblocking workflow](https://github.com/broadinstitute/warp/blob/master/pipelines/broad/dna_seq/germline/joint_genotyping/reblocking/ReblockGVCF.wdl) calls the GATK ReblockGVCF tool and uses the arguments:
```

Comment:
I think there was an earlier version of this PR that did call reblocking as a subworkflow, but as it is, the single-sample pipelines and reblocking pipeline just point at the same task.